plugin/trino-hive/src/main/java/io/trino/plugin/hive/HiveMetastoreClosure.java

```diff
@@ -177,9 +177,9 @@ public void createDatabase(Database database)
         delegate.createDatabase(database);
     }
 
-    public void dropDatabase(String databaseName, boolean deleteData)
+    public void dropDatabase(String databaseName, boolean cascade, boolean deleteData)
```

Comment:
Is the order of the parameter intentionally different from the one used in `delegate`?
(`cascade` comes before `deleteData` in the method declaration)

I'd rather keep it similar to `io.trino.plugin.hive.metastore.HiveMetastore#dropDatabase`